lib/fog/libvirt/models/compute/templates/volume.xml.erb

```diff
@@ -20,8 +20,12 @@
           <path><%= backing_volume.path %></path>
           <format type="<%= backing_volume.format_type %>"/>
           <permissions>
+            <% if owner -%>
             <owner><%= owner %></owner>
+            <% end -%>
+            <% if group -%>
             <group><%= group %></group>
+            <% end -%>
             <mode>0744</mode>
```

Comment:
Unrelated, but what a weird mode.

Reply:
Indeed but it looks like this is how libvirt creates it by default. This is an image created by vagrant-libvirt:

```
-rwxr--r--. 1 lzap lzap  1735852032 23. pro 11.44 centos-VAGRANTSLASH-stream8_vagrant_box_image_20210210.0.img
```

Well, unless it also uses fog library which I don't think is the case, nothing in their Gemfile.

Reply:
> Well, unless it also uses fog library which I don't think is the case, nothing in their Gemfile.
That's because it's in the gemspec:
https://github.com/vagrant-libvirt/vagrant-libvirt/blob/b3cf54a946600329d691f81cce32d7c407f8d88e/vagrant-libvirt.gemspec#L27